vue-training/src/views/index/index.pug

```diff
@@ -0,0 +1,79 @@
+doctype html
+html(lang='en')
+  head
+    title Developer Tools
+    meta(charset='UTF-8')
+    meta(name='viewport' content='width=device-width, initial-scale=1')
+    link(rel='preload' href='./index.js' as='script')
+    link(rel='preload' href='./vendor.js' as='script')
+    link(rel='preload' href='styles.index.css' as='style')
+    link(rel='stylesheet' href='styles.index.css')
+    script(src='./vendor.js')
+  body.app-container
+    nav.navbar
+      img.navbar-logo(src='assets/wolox_logo.svg' alt='App logo')
+      h1.navbar-text Developer Tools
+    #app.main(v-cloak)
+      h2.title Front-End Tools
+      .tech-box-list
+        .tech-box
+          img.tech-box-logo(src='assets/tech/react_icon.svg' alt='React logo')
+          h3.tech-box-title ReactJS
+          span.tech-box-launch Release: 2013
+          p.tech-box-text
+            | React is a declarative, component-based JavaScript library for building user interfaces.
+        .tech-box
+          img.tech-box-logo(src='assets/tech/angular_icon.svg' alt='AngularJS logo')
+          h3.tech-box-title AngularJS
+          span.tech-box-launch Release: 2010
+          p.tech-box-text
+            | AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architecture.
+        .tech-box
+          img.tech-box-logo(src='assets/tech/vue_icon.svg' alt='VueJS logo')
+          h3.tech-box-title VueJS
+          span.tech-box-launch Release: 2014
+          p.tech-box-text
+            | Vue.js (pronounced /vju&#x2D0;/, like view) is an open-source JavaScript framework for building user interfaces.
+        .tech-box
+          img.tech-box-logo(src='assets/tech/elm_icon.svg' alt='Elm logo')
```

Comment:
remove logo alt

Reply:
Eliminated all attributes 'alt' but, why?

Reply:
Only the logo word, as it's repeated it gives no additional info to the image

Reply:
Do not delete the whole alt 😅

Reply:
ups, sorry :stuck_out_tongue_winking_eye: